INDEX

```diff
@@ -493,6 +485,14 @@
         "category": "security",
         "homepage": "https://zoneminder.com/"
     },
+    "zrepl": {
+        "MANIFEST": "zrepl.json",
+        "description": "zrepl is a one-stop, integrated solution for ZFS replication.",
+        "icon": "https://icons.freenas.org/community-icons/zrepl.png",
+        "name": "zrepl",
+        "official": false,
+        "primary_pkg": "zrepl"
+    },
     "zwavejs2mqtt": {
         "MANIFEST": "zwavejs2mqtt.json",
         "name": "Z-WaveJS to MQTT",
```

Comment:
There is a missing white space here. that has already been corrected and merged 

https://github.com/ix-plugin-hub/iocage-plugin-index/commit/6ed79f28880a17981ab2e323474452818685cfbe

Reply:
Missing white-space... where ?

Reply:
No if you click on the link it clearly shows what I'm talking about...

Again I just happen to notice this change had not been applied here - I have no idea if this file will overwrite that commit or if it will persist - So I wanted to ask

Reply:
Before finishing here, I really need some help [here](https://github.com/zorglube/iocage-plugin-movienight), I tryed a lot of solutions and the Jail creation always fail : 
```
movienight_1 had a failure
Exception: SystemExit Message: 1
Partial plugin destroyed
```

Reply:
@zorglube you can use `iocage fetch -k` flag for keeping the malformed jail there even after a failure to install it to investigate what went wrong. Let me know please once the PR is ready for review. Thank you

Reply:
Thank for the tip. 

I have a big PR from @tprelog to read and understand.